packages/app-shell/src/components/Toasts/PopToast.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect } from 'react';
+import { Props as BaseProps } from '@airbnb/lunar/lib/components/Toast';
+import { Omit } from 'utility-types';
+import AppContext from '../AppContext';
+
+export type Props = Omit<BaseProps, 'id' | 'onRemove'>;
+
+export default function PopToast({ message, ...props }: Props) {
+  const context = useContext(AppContext);
+
+  if (!context) {
+    return null;
+  }
+
+  useEffect(() => {
+    if (props.danger || message instanceof Error) {
+      context.addFailureToast(message, props);
+    } else if (props.refresh) {
+      context.addRefreshToast(message, props);
+    } else if (props.success) {
+      context.addSuccessToast(message, props);
+    } else {
+      context.addInfoToast(message, props);
+    }
+  }, [message]);
```

Comment:
same q about adding `props` here for `props.danger`, `props.refresh`, and `props.success`